Agent/General/NewRelicAgentInternal.m

```diff
@@ -149,6 +149,17 @@ - (id) initWithCollectorAddress:(NSString*)collectorHost
 
     self = [super init];
     if (self) {
+
+        if ([NRMAFlags shouldEnableBackgroundReporting]) {
+            if (@available(iOS 13.0, *)) {
+                [[BGTaskScheduler sharedScheduler] registerForTaskWithIdentifier:[[NSBundle mainBundle] bundleIdentifier] usingQueue:nil launchHandler:^(__kindof BGTask * _Nonnull task) {
+                    [self handleAppRefreshTask: task];
+                }];
+            } else {
+                // Fallback on earlier versions
```

Comment:
Is there an actual fallback for pre-iOS 13 systems? Or do we just not support this feature on those?

Reply:
BackgroundReporting is only enabled for iOS 13+. Will note this.

Reply:
In that case, I would just get rid of all the else clauses. There's no need for them.

Reply:
Done